style.css

```diff
@@ -19,14 +19,20 @@
 .main {color: aliceblue}
 
 .link {
-    color: aliceblue;
+    color: aliceblue; 
 
-    background: antiquewhite;
-  color: aqua;
+    background: antiquewhite; 
+  color: aqua; 
 
 
 
 
 
       position:relative;
-}
\ No newline at end of file
+  position:relative;
+}
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/test_test/pullRequest?prid=5457187)